Makefile

```diff
@@ -18,13 +18,15 @@ ROOT_DIR_RELATIVE := .
 
 include $(ROOT_DIR_RELATIVE)/common.mk
 
-GO_VERSION ?= 1.20.12
```

Comment:
@kishen-v why this change?

Reply:
Sorry, it was an unintended change which went along with latest commit. 
I have reverted back to the version on `main`.